cves/CVE-2004-0488.yml

```
@@ -31,7 +31,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: This issue with this CVE was a buffer overflow in Apache HTTP Server 2.0.50. By using a user with trusted CA (Certificate Authority) an remote attacker could
```

```suggestion
description: This issue with this CVE was a buffer overflow in Apache HTTP Server 2.0.50. By using a user with trusted CA (Certificate Authority) a remote attacker could
```
sorry if this is a bit grammar nazi tier but good grammar makes it easier to read these types of things

---

Good find, and has been fixed.